test/integration/full/target-size/shadow-dom.js

```diff
@@ -0,0 +1,45 @@
+describe('target-size shadow dom test', () => {
+  'use strict';
+  let results;
+
+  before(done => {
+    axe.testUtils.awaitNestedLoad(() => {
+      const options = {
+        runOnly: ['target-size'],
+        elementRef: true
+      };
+      const context = {
+        // ignore the mocha links
+        exclude: '#mocha'
+      };
+      axe.run(context, options, (err, r) => {
```

Comment:
Any reason you're not using await here?

Reply:
just copy pasta from all the other test files